src/components/screens/wallet/delegatePerformance/delegatePerformance.js

```diff
@@ -33,28 +33,34 @@ const DelegatePerformance = ({ delegate: { error, isLoading, data } } = {}) => {
         <Box className={grid.row}>
           <Box className={`${grid['col-md-12']} ${grid['col-xs-12']}`}>
             <p className={styles.description}>
-              {delegatePerformanceDetails(pomHeights.length, status, consecutiveMissedBlocks)}
+              {pomHeights?.length
+                ? delegatePerformanceDetails(pomHeights.length, status, consecutiveMissedBlocks)
+                : 'The delegate was banned after 5 punishments.'}
```

Comment:
since status `The delegate is permanently banned from generating blocks due to repeated protocol violations or missing too many blocks.` itself is very much clear and we do not have details to show. We can remove the details link and keep just the status?

Reply:
@ManuGowda Resolved